testing.py

```
@@ -19,15 +19,51 @@
 
 ## API Reference
 #The PMLL API is designed to be easy to use and flexible. Here are some examples:
-
-* `pml.init()`: Initializes the PMLL
-* `pml.train()`: Trains the PMLL on a dataset
-* `pml.predict()`: Makes predictions using the PMLL
+__package__= "pml"
+import xml
+import pml # type: ignore
+
+xml.init();
```

Ruff (0.8.2)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)